src/common/lib/uart.c

```
@@ -368,30 +373,30 @@ static void start_next_dma_if_needed() {
  * continuous output without CPU intervention.
- * 
+ *
  * The handler is designed to be as fast as possible to minimize interrupt
  * latency impact on real-time operations. It uses low-level register access
  * and efficient queue operations.
- * 
+ *
  * Handler Priority: Low (configured in init_uart_dma)
  * Execution Context: Interrupt (keep minimal and fast)
  */
 void __isr dma_handler() {
```

Clang (14.0.6)

[warning] declaration uses identifier '__isr', which is a reserved identifier

(bugprone-reserved-identifier)